src/components/annotations/convert_coords.js

```diff
@@ -26,6 +26,8 @@ var toLogRange = require('../../lib/to_log_range');
  *     same relayout call should override this conversion.
  */
 module.exports = function convertCoords(gd, ax, newType, doExtra) {
+    ax = ax || {};
```

Comment:
Man, if only I could turn back time and not use linearized coordinates on log axes...